app/templates/index.html

```diff
@@ -0,0 +1,6 @@
+{% extends "base.html" %}
+{% block content %}
+{% for book in books.values() %}

```

Comment:
it would be nice if you use some visual indentation, otherwise it is hard to see the structure